README.md

```diff
@@ -238,7 +238,7 @@ template = keynodes['my_template']
 search_results = client.template_search(template)
 search_result = search_results[0]
 
-search_result.size()
+len(search_result)
```

Comment:
Don't remove this logic. Leave it and add your own one if it is matter

Reply:
In versions 0.2.X I won't remove it

Reply:
But documentation have been changed. And tests so have been changed

Reply:
I'll add deprecated in docs

Reply:
Added. Also I think we need refactor docs, there is no everything